groups/groups.yaml

```diff
@@ -571,6 +571,15 @@ groups:
       - ha.chuck@gmail.com
       - hi@amycod.es
       - prignanov@vmware.com
+  
+  - email-id: k8s-infra-staging-cluster-api-do@kubernetes.io
+    name: k8s-infra-staging-cluster-api-do
+    description: |-
+      ACL for staging CAPI for DigitalOcean
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - mudrinic.mare@gmail.com
```

Comment:
Right now, I'm the only one maintainer of the CAPI-DO project ([SCL mailing list post for more details](https://groups.google.com/forum/#!topic/kubernetes-sig-cluster-lifecycle/iKvpeKAJmNo)). We will add two new maintainers after the lazy consensus passes on 3/25. 

Is it okay that I'm the only member/owner for now? Should I wait for new maintainers to get promoted or add someone from SCL/Cluster-API?

Reply:
A single owner/member here should be fine for now, I'm also more than happy to be a backup resource to assist if needed as well.